package-lock.json

```diff
@@ -1,8 +1,3785 @@
 {
   "name": "audit-ci",
   "version": "3.1.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This project still technically supports Node 8, which is way behind the Node 15 that NPM 7 is typically used. Will this bring friction to newcomers? It's not documented that this project would use NPM 7. My instinct is that this should stay v1.

Reply:
Reverted